Lib/test/test_codecs.py

```diff
@@ -3439,6 +3439,23 @@ def search_function(encoding):
         self.assertEqual(NOT_FOUND, codecs.lookup('BBB.8'))
         self.assertEqual(NOT_FOUND, codecs.lookup('a\xe9\u20ac-8'))
 
+    def test_encodings_normalize_encoding(self):
+        # encodings.normalize_encoding() ignores non-ASCII letters.
+        out = encodings.normalize_encoding('utf_8')
```

Comment:
Would it be possible to avoid the "out" temporary variable in tests? 

Something like `self.assertEqual(encodings.normalize_encoding('utf_8'), 'utf_8')` but indent it properly to avoid long lines (I like to restrict myself to 80 columns).

Reply:
no problem, updated.